spec/models/comment_spec.rb

```diff
@@ -3,6 +3,17 @@
 describe Comment do
   it_behaves_like "email_confirmable"
 
+  describe "#confirm!" do
+    let(:comment) do
+      VCR.use_cassette('planningalerts') { build(:comment, confirmed: true) }
+    end
+
+    it "should run callback once already confirmed" do
```

Comment:
I think I get what this is about. I think a context, like "when already confirmed", might make this clearer.